aunt-leahs-app/src/pages/VolunteerCheckInPage.jsx

```diff
@@ -15,20 +17,28 @@ import moment from 'moment';
 function VolunteerCheckInPage({ location, volunteer }) {
 
 	const [duration, setDuration] = useState();
+	const [isFormSubmitted, setIsFormSubmitted] = useState(false);
+	const [isSubmissionSuccessful, setIsSubmissionSuccessful] = useState(false);
 
 	const postShift = async () => {
 		try {
-			console.log(volunteer);
-		const res = await axios.post('http://localhost:7071/api/shifts', {
-			shiftData: {
-				locationId: location.id,
-				volunteerId: volunteer.id,
-				startTime: now,
-				duration: duration
-			}});
-		
-			console.log(`statusCode: ${res.statusCode}`);
-			console.log(res);
+
+			const res = await axios.post('http://localhost:7071/api/shifts', {
+				shiftData: {
+					locationId: location.id,
+					volunteerId: volunteer.id,
+					startTime: now,
+					duration: duration
+				}
+			});
+
+			if (res.status === '200') {
```

Comment:
Does axios really return a string for the status? Looks strange to me but if you've tested it and it works then it's good

Reply:
You're right, the status is a number. This still worked for some reason though... I will change it nonetheless.

Reply:
Made the change